setup.json

```diff
@@ -3,7 +3,7 @@
         "appdirs",
         "argcomplete",
         "cryptography",
-        "jwt",
+        "jwt==0.3.2",
         "gitpython",
```

Comment:
It does, but that is handled in the provisioning at the moment. 

Since this commit is literally what is in production I would like to merge this, tag it and then fix and clean that up in a future PR.